server/api_server/api_server_start.py

```
@@ -444,6 +445,62 @@ def api_device_update_column(mac, payload=None):
+        
+        if result.get("success"):
+            action = "locked" if should_lock else "unlocked"
+            return jsonify({
+                "success": True, 
+                "message": f"Field {field_name} {action}",
+                "fieldName": field_name,
+                "locked": should_lock
+            })
+        else:
+            return jsonify(result), 400
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)